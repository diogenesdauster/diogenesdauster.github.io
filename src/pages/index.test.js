import React from 'react'
import renderer from 'react-test-renderer'
import IndexPage from './index'

/* eslint-disable */
describe('IndexPage', () =>
  it('renders correctly', () => {
    const tree = renderer
      .create(<IndexPage data={indexData} layoutData={sideData} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  }))
/* eslint-disable */
