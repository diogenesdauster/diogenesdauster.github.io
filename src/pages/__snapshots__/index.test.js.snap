// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage renders correctly 1`] = `
Array [
  <div
    className="sidebar"
  >
    <div
      className="container sidebar-sticky"
    >
      <div
        className="sidebar-about"
      >
        <h1>
          <a
            href="/"
          >
            <span
              aria-label="emoji"
              role="img"
            >
              üë®üèª‚Äçüíª
               
            </span>
            <span
              className="tags"
            >
              &lt;/
            </span>
            D.Dauster
            <span
              className="tags"
            >
              &gt;
            </span>
          </a>
        </h1>
        <p
          className="tagline"
        >
          Code Has No Accent
        </p>
      </div>
      <input
        id="menu-icon"
        type="checkbox"
      />
      <label
        className="menu-label"
        htmlFor="menu-icon"
      />
      <div
        className="trigger"
      >
        <ul
          className="sidebar-nav"
        >
          <li>
            <Link
              className="sidebar-nav-item"
              to="/"
            >
              Home
            </Link>
          </li>
          <li>
            <Link
              className="sidebar-nav-item"
              to="about"
            >
              About
            </Link>
          </li>
          <li>
            <Link
              className="sidebar-nav-item"
              to="archive"
            >
              Archive
            </Link>
          </li>
          <li>
            <Link
              className="sidebar-nav-item"
              to="projects"
            >
              Projects
            </Link>
          </li>
        </ul>
      </div>
      <p
        className="copyright"
      >
         
        ¬© 
        2018
        . All rights reserved.
      </p>
    </div>
  </div>,
  <div
    className="content container"
  >
    <div
      className="posts"
    >
      <div
        className="post"
      >
        <h1
          className="post-title"
        >
          <Link
            to="/2013/12/28/introducing-hyde"
          >
            Introducing Hyde
          </Link>
        </h1>
        <span
          className="post-date"
        >
          December 28th, 2013
        </span>
        <p>
          Hyde is a brazen two-column  Jekyll  theme that pairs a prominent sidebar with uncomplicated content. It's based on  Poole , the Jekyll‚Ä¶
        </p>
      </div>
      <div
        className="post"
      >
        <h1
          className="post-title"
        >
          <Link
            to="/2012/02/06/whats-jekyll"
          >
            What's Jekyll?
          </Link>
        </h1>
        <span
          className="post-date"
        >
          February 6th, 2012
        </span>
        <p>
          Jekyll  is a static site generator, an open-source tool for creating simple yet powerful websites of all shapes and sizes. From  the project‚Ä¶
        </p>
      </div>
      <div
        className="post"
      >
        <h1
          className="post-title"
        >
          <Link
            to="/2012/02/07/example-content"
          >
            Example content
          </Link>
        </h1>
        <span
          className="post-date"
        >
          February 7th, 2012
        </span>
        <p>
          Cum sociis natoque penatibus et magnis  dis parturient montes , nascetur ridiculus mus.  Aenean eu leo quam.  Pellentesque ornare sem‚Ä¶
        </p>
      </div>
    </div>
  </div>,
]
`;
